TestSystemBackend Failure in Vault 1.11

Vault 1.11 introduced the following errors in test_system_backend.py. Creating an issue to track the resolution of these failures.

```
FAILED tests/integration_tests/v1/test_system_backend.py::TestSystemBackend::test_start_generate_root_then_cancel - hvac.exceptions.InvalidRequest: OTP string is wrong length, on put https://localhost:8200/v1/sys/generate-root/attempt
FAILED tests/integration_tests/v1/test_system_backend.py::TestSystemBackend::test_start_generate_root_with_completion - hvac.exceptions.InvalidRequest: OTP string is wrong length, on put https://localhost:8200/v1/sys/generate-root/attempt
```

--- a/tests/utils/__init__.py
+++ b/tests/utils/__init__.py
@@ -70,7 +70,9 @@ def get_generate_root_otp():
     :return: OTP to use in generate root operations
     :rtype: str
     """
-    if vault_version_ge("1.0.0"):
+    if vault_version_ge("1.10.0"):
+        test_otp = "BMjzW3wAsEzINXCM05Wbas3u9zSl"
+    elif vault_version_ge("1.0.0"):
         test_otp = "ygs0vL8GIxu0AjRVEmJ5jLCVq8"
     else:
         test_otp = "RSMGkAqBH5WnVLrDTbZ+UQ=="

